Credify/Credify/Objects/serviceXConfig.swift

```diff
@@ -1,29 +1,38 @@
 //
-//  File.swift
+//  serviceXConfig.swift
 //  
 //
 //  Created by Shu on 2022/03/12.
 //
 
 import Foundation
 
+
+/// This is serviceX configiration
 public struct serviceXConfig {
     let apiKey: String
     let env: CredifyEnvs
     let appName: String
     let theme: serviceXTheme
     let userAgent: String
     
+    
+    /// This is constructor to create serviceX configuration
+    /// - Parameters:
+    ///   - apiKey: Your API that gets from the Dashboard
```

Comment:
`API key that you can generate on the Dashboard`

Reply:
I updated it